src/OpenTK.Windowing.Common/OpenTK.Windowing.Common.csproj

```diff
@@ -19,6 +19,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OpenTK.Core\OpenTK.Core.csproj" />
+    <ProjectReference Include="..\OpenTK.GraphicsLibraryFramework\OpenTK.GraphicsLibraryFramework.csproj" />
```

Comment:
The common library shouldn't reference GLFW.

Reply:
Copy-paste error. Fixed it.